en/lessons/specifics/ecto/02_associations.md

```diff
@@ -0,0 +1,672 @@
+---
+version: 1.3.0
+title: Associations
+---
+
+In this section we'll learn how to use Ecto to define and work with associations between our schemas.
+
+## Table of Contents
+
+* Set up
+* Types of associations
+  * Belongs to/has many
+  * Belongs to/has one
+  * Many to Many
+* Saving associated data
+  * `build_assoc`
```

Comment:
I'm not sure if `build_assoc` vs. `put_assoc` is the right way to outline it. `put_assoc` is for any relationships that implement a `belongs_to` and `put_assoc` is for the `many_to_many`. `put_assoc` is not the only way  to save such data. You can also use `cast` assoc. I didn't include anything on cast assoc here since I felt like it was running a little long/would have required more of a deep dive but I'm open to including it. 

So with this in mind, might make more sense to outline more like:
* Saving data with `belongs_to`
* Saving data with `put_assoc`

Reply:
I think this makes sense unless I'm missing something myself -- feel free to rock it out in that direction